docs/issue_template.md

```diff
@@ -1,14 +1,18 @@
-<!--- The problem --->
-1. Replace this text with a clear statement of the problem that needs to be solved including any relevant supporting files.
-- **If its a software abnormality**, explain how it should act, how it does act, and how to reproduce it.
-- **If you are reporting a security related problem,** DO NOT report here. Email [support@daemo.org](mailto:support@daemo.org) instead.
-- **If this is associated with a strategic milestone** add the `strategy` milestone.
-2. Add a title in the field above that concisely summarizes the problem.
-
-## My proposal
-3. If you have a solution to this problem and you will implement it, continue, otherwise remove the **my proposal** section.
+1. Add a proposal title above that concisely summarizes the problem or suggestion.
+
+## Problem
+2. Replace this text with a clear statement of the problem that needs to be solved. If feasible, include specific evidence or examples (e.g., screen shots, data) in support of the understanding of the problem. If this is a bug, report it in the Daemo repository instead of here: https://github.com/crowdresearch/daemo/issues/new
+
+## Proposal
 4. Add yourself as an assignee to this proposal.
-5. Replace this text with a clear statement of the solution you propose to implement, including any relevant supporting files. It can strengthen your proposal to include *pros* and *cons* and your thoughts about the *implications* (short and long term) of it being executed.
+5. If this is a strategic milestone, add the `strategy` milestone.
+6. If you have a solution to this problem and you will implement it, replace this text with a clear statement of the solution you propose to implement, including any relevant supporting files. It can strengthen your proposal to include *pros* and *cons*. If you have no solution, remove this Proposal section.
```

Comment:
I think, ideally this should go before 4 as it is the first thing to do when deciding to make this a structured proposal instead of just reporting a problem.